ubuntu/debian/tests/build

```diff
@@ -76,8 +76,8 @@ int main(int argc, char **argv)
 }
 EOF
 
-g++ -o igntest igntest.c `pkg-config --cflags --libs ignition-math7`
+g++ -o gztest gztest.c `pkg-config --cflags --libs gz-math7`
```

Comment:
I believe this line requires https://github.com/gazebosim/gz-math/pull/420 to be merged first